python/ray/serve/examples/doc/snippet_model_ensemble.py

```diff
@@ -0,0 +1,57 @@
+import ray
```

Comment:
Please add the following to our `BUILD` file so the snippet is covered by our CI to ensure it's always executable, which is the ideal standard we want for our external user facing documentations:

```
py_test(
    name = "snippet_model_ensemble",
    size = "small",
    srcs = glob(["examples/doc/*.py"]),
    tags = ["exclusive", "team:serve"],
    deps = [":serve_lib"]
)

```

Reply:
yes sure, added it.